website/docs/jellyfish/api/account.md

```diff
@@ -162,3 +162,23 @@ interface UtxosToAccountUTXO {
   vout: number
 }
 ```
+
+## accountToAccount
+
+Creates and submits to a local node and transfer transaction from the specified account to the specfied accounts.
+Optionally, specific UTXOs to spend to create that transaction.
+
+```ts title="client.account.accountToAccount()"
+interface account {
+  accountToAccount (from: string, payload: AccountToAccountPayload, utxos: AccountToAccountUTXO[] = []): Promise<string>
+}
+
+interface AccountToAccountPayload {
+  [key: string]: string;
```

Comment:
@jingyi2811 for this payload, you can use a template literal type. Regex is not yet available for TS but you can specify a custom type. Example,
![image](https://user-images.githubusercontent.com/25013382/119425043-37b00480-bd39-11eb-8c84-0e09c4b3240f.png)

Having this type assures as to have the proper format instead of just having plain string.

Reply:
I am unsure of the best way to adopt this in my latest code. It is a good suggestion anyway. Probably we can add this type as a global setting somewhere.